### PR TITLE
Make active navigation indicator more obvious

### PR DESCRIPTION
**Pull request by juliusknorr**
Increase the width of the active navigation item to 5px and also use it for entries in the second hierarchy level.

Fixes https://github.com/nextcloud/server/issues/4627

bildschirmfoto vom 2018-03-19 17-04-03

@nextcloud/designers

### CODE REVIEW

File: core/css/apps.scss [hunk 1]
```diff
@@ -197,6 +197,13 @@ kbd {
 						}
 					}
 
+					&.active {
+						box-shadow: inset 5px 0 $color-primary;
+						left: -44px;
```
Comment (skjnldsv) [inline core/css/apps.scss]: Not quite fan of those minus dirty hacks! :/
I was pretty happy on how we successfully have a working structure without (many) hacks on negative margins and stuff. Isn't there any other alternatives?

Comment (juliushaertl) [reply core/css/apps.scss]: Do we have nested lists deeper than 2? If not we could just apply the padding for the nesting indentation to the li-element and get rid of the negative offset.

Comment (skjnldsv) [reply core/css/apps.scss]: Nope, only one max allowed! :)

File: core/css/apps.scss [hunk 2]
```diff
@@ -152,7 +152,7 @@ kbd {
 				&,
 				> a {
 					opacity: 1;
-					box-shadow: inset 2px 0 $color-primary;
+					box-shadow: inset 5px 0 $color-primary;
```
Comment (MorrisJobke) [inline core/css/apps.scss]: I would go for `4px` ... `5px` is a bit too heavy at least in my opinion.

Comment (juliushaertl) [reply core/css/apps.scss]: I agree, looks a bit nicer.

File: core/css/apps.scss (hunk 1 above)
Comment (juliushaertl) [reply core/css/apps.scss]: Awesome. Fixed that.

**Comment by jancborchardt** [comment]
Great! (I guess because of the way we do it, it's not possible to give it a 3px border-radius on top and bottom?)

**Review by MorrisJobke — APPROVED** [review]
:+1:

**Comment by MorrisJobke** [comment]
> (I guess because of the way we do it, it's not possible to give it a 3px border-radius on top and bottom?)

Correct. Should not be possible. :/

**Review by skjnldsv — APPROVED** [review]

**Review by skjnldsv — DISMISSED** [review]
The `app-navigation-entry-deleted` & `app-navigation-entry-edit` needs a left padding for sub-items :)

**Comment by skjnldsv** [comment]
For testing: 
[templatetest.tar.gz](https://github.com/nextcloud/server/files/1836623/templatetest.tar.gz)

**Comment by juliushaertl** [comment]
@skjnldsv Thanks, fixed as well.

**Review by MorrisJobke — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**